modules/monitoring-searching-alerts-silences-and-alerting-rules.adoc

```diff
@@ -15,10 +15,10 @@ In the *Administrator* perspective, the *Alerts* page in the Alerting UI provide
 
 You can filter by alert state, severity, and source. By default, only *Platform* alerts that are *Firing* are displayed. The following describes each alert filtering option:
 
-* *Alert State* filters:
```

Comment:
Was this line changed on purpose? I'd argue that `Alert state` is better the just `State`. below the edit is different and preserves `Alert`.

Reply:
Hello, @jan--f I made the change to correctly reflect the UI there, because there are actual sections called "Alert state" but here the section is just "State". It was to differentiate between the two. I hope the change makes more sense now. Let me know, what you think, thank you!

Reply:
Also, to give an example, here: https://github.com/openshift/openshift-docs/pull/72590/files#diff-a4936ced8c7560436649e29015992b60f559b68904004b23bc1d05a22540b314R53 the filters are actually called "Alert State", while the filter you pointed out is just called "State" filter.